src/suggestion.rs

```diff
@@ -37,7 +52,16 @@ pub enum Detector {
+/// pneumonoultramicroscopicsilicovolcanoconiose of food, what's up with pie? There's strawberry pie, apple, pumpkin and so many others, but there is no grape pie! I know. I'm just as upset about this unfortunate lack of development in the pie division. Think about it. Grapes are used to make jelly, jam, juice and raisins. What makes them undesirable for pie? Would they dry into raisins? Couldn't you just stick some jelly in a piecrust and bake it? It just doesn't make any sense. Another thing that bothers me is organ grinders. You know, the foreign guys with the bellhop hats and the little music thingy and the cute little monkey with the bellhop hat who collects the money? Okay. They're basically begging on the street. How did they ever afford an organ-thingy? Wouldn't it make more sense to get a kazoo, if you're broke? And if they're so poor, what possessed them to buy a monkey?
+/// Reasn of food, what's up with pie? There's strawberry pie, apple, pumpkin and so many others, but there is no grape pie! I know. I'm just as upset about this unfortunate lack of development in the pie division. Think about it. Grapes are used to make jelly, jam, juice and raisins. What makes them undesirable for pie? Would they dry into raisins? Couldn't you just stick some jelly in a piecrust and bake it? It just doesn't make any sense. Another thing that bothers me is organ grinders. You know, the foreign guys with the bellhop hats and the little music thingy and the cute little monkey with the bellhop hat who collects the money? Okay. They're basically begging on the street. How did they ever afford an organ-thingy? Wouldn't it make more sense to get a kazoo, if you're broke? And if they're so poor, what possessed them to buy a monkey?
+/// Speaking of food, what's up with pie? There's strawberry pie, apple, pumpkin and so many others, but therie is no grape pie! I know. I'm just as upset about this unfortunate lack of development in the pie division.
+/// Speaking of food, what's up with pie? There's strawberry pie, apple, pumpkin and so many others, but therieeeeeeeeeeeeeeee is no grape pie! I know. I'm just as upset about this unfortunate lack of development in the pie division.
+/// Speaking of food, what's up with pie? There's strawberry pie, apple, pumpkin and so many otherrrrrrrrrrrrrs, but therieeeeeeeeeeeeeeee is no gruuuuuuuuuuuuuuuuuuuuuuuape pie! I know. I'm just as upset about this unfortunate lack of development in the pie division.
```

Comment:
this should be removed. It is just to keep track of the examples that I have been using.

Reply:
Keep them in the commit history, we might want to create a file with test cases, such that one can create additional test cases at ease and those are used everywhere.